ovos_gui_plugin_shell_companion/__init__.py

```diff
@@ -155,11 +158,17 @@ def handle_device_about_page(self, message):
         self.gui['system_info'] = system_information
         self.gui.show_page("SYSTEM_AdditionalSettings.qml", override_idle=True)
 
-    def handle_display_wallpaper_rotation_config_set(self, message):
+    def handle_display_wallpaper_rotation_config_set(self, message: GUIMessage):
+        """
+        Handle a GUI event requesting a new wallpaper rotation setting
+        @param message: `speaker.extension.display.set.wallpaper.rotation`
+        """
         wallpaper_rotation = message.data.get("wallpaper_rotation", False)
         self.local_display_config["wallpaper_rotation"] = wallpaper_rotation
         self.local_display_config.store()
-        self.bus.emit(Message("speaker.extension.display.wallpaper.rotation.changed"))
+
+        # TODO Is the below message consumed anywhere?
```

Comment:
i don't think so, a org wide github search doesn't show it anywhere else, can be safely removed